src/components/Class/index.js

```diff
@@ -67,10 +65,40 @@ function Class() {
       payload[TIME_CONSTANT.CLASS_END_DATE] = `${moment(classEndTime).format(
```

Comment:
@komalng can you delete the redux code that has the create class api as well?

Reply:
https://github.com/navgurukul/bhanwari-devi/pull/81/commits/34a7b90788c3968197dbb480953e03159cce28a7